client/src/components/Questions/QuestionWithAnswer.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+function QuestionWithAnswer({ questionAsked }) {
+  const { question } = questionAsked;
```

Comment:
nit: take advantage of destructuring and try to combine multiple variables into one line (unless it was too long that ES lint complained)

Reply:
Great call and I applied this to all files where I was similarly doing this on separate lines.